app/views/categories/new.html.erb

```diff
@@ -1,10 +1,9 @@
-<%= form_with model: @category do |f| %>
+<%= form_with model: @category, class: 'new-form' do |f| %>
   <div class='field'>
-    <%= f.label :name %>
-    <%= f.text_field :name %>
+    <%= f.text_field :name, placeholder: 'Name' %>
   </div>
   
   <div class='field'>
-    <%= f.submit 'Add' %>
+    <%= f.submit 'Add', class: 'add-button' %>
```

Comment:
- [ ]  As stated in the project requirement Kindly make sure that For "Add a new category" page **The user fills out a form to create a new category, indicating their **name** and **icon** (both mandatory)** 
- But Your "Add a new category" page Doesn't have Icon input to create a form 
![image](https://github.com/Movgto/ror-budget-app/assets/93598189/d2a94447-ed06-4b45-9e04-64ebcd0b63e4)

- [ ]  As stated in the project requirement Kindly make sure that For "Add a new category" page Kindly make sure that **When the user clicks on the "Back" button (<), the user navigates to the home page**.
![image](https://github.com/Movgto/ror-budget-app/assets/93598189/c19ee19a-931a-4a9f-80bf-a9543101dc31)